### DISCUSSION TITLE
feat: configurable font settings for terminal

### DISCUSSION THREAD

**Discussion by taciturnaxolotl** [zed-industries/zed#7160] [Feature Requests]
It would be very helpful to be able to configure the terminal font settings separately from the buffer. For example, my edit font is Monaspace Neon Var, but for my terminal I have a nerd font for extra icons.

**Comment by osiewicz** [comment]
Hey, thanks for reaching out 
You can set font settings in `terminal` section of settings, e.g:
```
  "terminal": {
    "dock": "bottom",
    "font_family": "Comic Sans MS"
  },
```

**Reply by marcelo-albuquerque** [reply]
The option "font_family" is not in documentation?

**Reply by taciturnaxolotl** [reply]
It was disabled in the example settings, and I didn't know the documentation existed; I was just going off the observation that the terminal font was changed when changing the buffer font settings.

**Comment by taciturnaxolotl** [comment]
Thanks works perfectly!

**Comment by lmil** [comment]
thanks!
